command/meta.go

```diff
@@ -210,6 +210,18 @@ func (m *meta) requestUserApprovalDelete(taskName string) (int, bool) {
 	return m.requestUserApproval(taskName, "deleting")
 }
 
+// requestUserApprovalCreate prints a prompt for user approval of deleting a task
+// and waits for the user input. It returns an exit code and boolean describing
+// if the user approved.
+func (m *meta) requestUserApprovalCreate(taskName string) (int, bool) {
+	m.UI.Info("Creating the task will perform the actions described above.")
+	m.UI.Output(fmt.Sprintf("Do you want to perform these actions for '%s'?", taskName))
+	m.UI.Output(" - This action cannot be undone.")
+	m.UI.Output(" - Consul Terraform Sync cannot guarantee Terraform will perform")
```

Comment:
I know you didn't introduce this, but I think it's worthwhile to rename (here and others) to `Consul-Terraform-Sync` to match our documentation!

Reply:
Changed this and other instances in this file